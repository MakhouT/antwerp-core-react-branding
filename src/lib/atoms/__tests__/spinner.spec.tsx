import * as React from "react";

import * as A from "../..";
import { renderAndMatchSnapshot } from "../../../__tests__/__fixtures__/testUtils";

describe("Spinner", () => {
    it("Renders default", () => {
        // Arrange
        const element = <A.Spinner />;

        // Act and assert
        const tree = renderAndMatchSnapshot(element);
        expect(tree.props.className).toBe("a-spinner");
    });

    it("Renders small", () => {
        // Arrange
        const element = <A.Spinner small />;

        // Act and assert
        const tree = renderAndMatchSnapshot(element);
        expect(tree.props.className).toContain("a-spinner--sm");
    });
});
