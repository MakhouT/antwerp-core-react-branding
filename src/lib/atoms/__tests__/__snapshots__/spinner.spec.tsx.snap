// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Renders default 1`] = `
<div
  className="a-spinner"
/>
`;

exports[`Spinner Renders small 1`] = `
<div
  className="a-spinner a-spinner--sm"
/>
`;
